apps/events/viewsets.py

```diff
@@ -133,8 +133,12 @@ class RatingViewSet(viewsets.ModelViewSet):
     http_method_names = ["post", "get", "put", "delete"]
 
     def get_queryset(self):
-        event = self.get_object()
-        self.queryset = Rating.objects.filter(event=event, user=self.request.user)
+        if self.kwargs.get("rating_id"):
+            event = Event.objects.filter(id=self.kwargs["event_id"])
+            self.queryset = Rating.objects.filter(event=event, rating=self.kwargs["rating_id"])
+        else :
+            event = Event.objects.filter(id=self.kwargs["event_id"])
+            self.queryset = Rating.objects.filter(event=event, user=self.request.user)
```

Comment:
```suggestion
        event = Event.objects.filter(id=self.kwargs["event_id"])
        if self.kwargs.get("rating_id"):
            filters = {"rating": self.kwargs["rating_id"]}
        else :
            filters = {"user": self.request.user}
        
        self.queryset = Rating.objects.filter(event=event, **filters)
```

Reply:
Also, why do we use this `user=self.request.user` filter only when there is no rating_id?

Reply:
These are unnecessary block-code.  Don't need to take it. Not my area of responsibility. Concerning one I had thought getting ratings by authorized user only.